### DISCUSSION TITLE
Error 500 create a new egg

### DISCUSSION THREAD

**Discussion by csgo-s** [pterodactyl/panel#3090] [Support]
I have a problem when I want to create a new egg ,  gives an error of 500, even when creating a nest it gives an error 500

![p](https://user-images.githubusercontent.com/32447962/107853466-817c7b00-6e2b-11eb-91a5-d2d05cbc6fb4.gif)

**Comment by parkervcp** [comment]
Your uploaded image goes nowhere and is not exactly viewable.

**Reply by csgo-s** [reply]
> Your uploaded image goes nowhere and is not exactly viewable.

Edited

**Comment by Sharkblack3D** [comment]
I have the same problem please can you help me?

**Reply by trenutoo** [reply]
> I have the same problem please can you help me?

You'd have to post the panel logs to find out why. You most likely have database issues or didn't provide a valid author email during the panel setup.

`tail -n 100 /var/www/pterodactyl/storage/logs/laravel-$(date +%F).log | nc bin.ptdl.co 99`

Even better, join the [Discord server](https://discord.gg/pterodactyl), where you can get a faster support response.
